contrib/clojure-package/examples/infer/objectdetector/src/infer/objectdetector_example.clj

```diff
@@ -54,27 +56,30 @@
     :validate [check-valid-dir "Input directory not found"]]
    ["-h" "--help"]])
 
-(defn result->map [{:keys [class prob x-min y-min x-max y-max]}]
-  (hash-map
-   :label class
-   :confidence (int (* 100 prob))
-   :top-left [x-min y-min]
-   :bottom-right [x-max y-max]))
-
-(defn print-results [results]
-  (doseq [_r results]
-    (println (format "Class: %s Confidence=%s Coords=(%s, %s)"
-                     (_r :label)
-                     (_r :confidence)
-                     (_r :top-left)
-                     (_r :bottom-right)))))
-
 (defn process-results [images results output-dir]
-  (dotimes [i (count images)]
-    (let [image (nth images i) _results (map result->map (nth results i))]
-      (println "processing: " image)
-      (print-results _results)
-      (draw/draw-bounds image _results output-dir))))
+  (mapv (fn [image-path result]
+          (println "looking at image" image-path)
+          (println "result: " result)
+          (let [buf (ImageIO/read (new File image-path))
+                width (.getWidth buf)
+                height (.getHeight buf)
+                names (mapv :class result)
+                coords (mapv (fn [result]
+                               (-> result
+                                   (update :x-min #(* width %))
+                                   (update :x-max #(* width %))
+                                   (update :y-min #(* height %))
+                                   (update :y-max #(* height %))))
+                             result)
```

Comment:
Should we rename `result` here with `predictions` which is less generic?
Also `prediction` instead of `result` in the `mapv` function?